src/cmake.mk

```diff
@@ -8,8 +8,7 @@ $(PKG)_CHECKSUM := 92c83ad8a4fd6224cf6319a60b399854f55b38ebe9d297c942408b792b1a9
 $(PKG)_SUBDIR   := cmake-$($(PKG)_VERSION)
 $(PKG)_FILE     := cmake-$($(PKG)_VERSION).tar.gz
 $(PKG)_URL      := http://www.cmake.org/files/v$(call SHORT_PKG_VERSION,$(PKG))/$($(PKG)_FILE)
-$(PKG)_TARGETS  := $(BUILD)
-$(PKG)_DEPS     :=
+$(PKG)_TARGETS  := $(BUILD) $(MXE_TARGETS)
```

Comment:
> Why do you need this change?

So right before the vtk7 package the error triggered: 
```make: *** No rule to make target '/home/sws/projSik/2016/01_build_vtk7/mxe/usr/i686-w64-mingw32.static/installed/cmake', needed by '/home/sws/projSik/2016/01_build_vtk7/mxe/usr/i686-w64-mingw32.static/installed/vtk7'.  Stop.```

This change made it build, but I don't know what it does.